packages/bigtable/src/table.js

```diff
@@ -659,7 +677,17 @@ Table.prototype.getRows = function(options, callback) {
  *   }
  * ];
  *
- * table.insert(entries, function(err, apiResponse) {});
+ * table.insert(entries, callback);
+ *
+ * //-
+ * // A stream mode is also provided, this is useful when making a large number
```

Comment:
We don't - I don't believe it's possible to stream entries to the backend. If we were to write a method in the library it would essentially buffer all the entries and then send them all at once.